graphics/rendering/blender/README.md

```diff
@@ -13,36 +13,42 @@ In this example, we will install Blender in a [Ubuntu container](https://cloud.s
 
 <br>
 
-### To start, Make the working directory:
+## To start, Make the working directory:
 
 ```
 $ mkdir ~/blender
 $ cd ~/blender/
 ```
 
+We will run this example in the `~/blender/` directory, not necessary but its cleaner this way.
```

Comment:
not necessary but **it's** cleaner this way.

Reply:
:+1: